### PR TITLE
NR-31187 Typescript Integration

### PR DESCRIPTION
**Pull request by pkosanam**
# Summary
Ticket URL: https://issues.newrelic.com/browse/NR-31187
Description: 
This PR has the new quickstart integration named "Typescript"

A concise description of the changes being introduced. If you are submitting a PR for a
new quickstart please review the pre merge checklist section to validate this PR is ready for review.

## Pre merge checklist

- [ ] Did you check you NRQL syntax? - Does it work?
- [ ] Did you check your dashboard image quality? -  Do they look good?
- [ ] Did you check that your alerts actually work?
- [ ] Did you include an InstallPlan and Documentation reference?
- [ ] Did you check your descriptive content for voice, tone, spelling and grammar errors?
- [ ] Did you attach images of your dashboards to the PR so we can see them working?

### Screenshots

Attach images of any visual changes, such as a dashboard here.

# Preview:
screencapture-newrelic-instant-observability-preview-2022-08-15-06_18_23

### CODE REVIEW

File: quickstarts/node-js/typescript/config.yml [hunk 1]
```diff
@@ -0,0 +1,66 @@
+
+slug: typescript
+description: |
+  ## Comprehensive monitoring quickstart for TypeScript
```
Comment (akristen) [inline quickstarts/node-js/typescript/config.yml:4]:
```suggestion
  ## Quickstart for comprehensive monitoring with TypeScript
```

File: dashboards/typescript/typescript.json [hunk 2]
```diff
@@ -0,0 +1,694 @@
+{
```
Comment (aswanson-nr) [inline dashboards/typescript/typescript.json:1]: I'm seeing a lot of entity GUIDs and explicit time ranges in these NRQL queries. Let's get this updated to handling entities with different GUIDs and remove the explicit time ranges.

**Review by aswanson-nr — CHANGES_REQUESTED** [review]
Quickstart looks good! However, seeing lots of unique GUIDs in the dashboard NRQL queries. Let's get those cleaned up before we merge.

**Comment by pkosanam** [comment]
Closing this PR as we have created a new PR related to typescript: https://github.com/newrelic/newrelic-quickstarts/pull/1356

**Pull request closed without merging**